src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="link"
+      method="get"
+      class="form"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        class="box box__big"
+        name="bigBox"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form
+      action="link"
+      method="get"
+      class="form form--small"
+      data-qa="small"
+    >
+      <input
+        type="text"
+        class="box box__small"
```

Comment:
```suggestion
        class="box box--small"
```
As I see, `box--small` and `box--big` are modifiers but not elements. So for modifiers you have to use `--` (minus-minus)